sandbox/settings.py

```diff
@@ -197,6 +197,18 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
     # Mapping between edx and richie profile fields
     EDX_USER_PROFILE_TO_DJANGO = values.DictValue()
 
+    # Feature flags
+    FEATURES = values.DictValue(environ_name="FEATURES", environ_prefix=None)
+
+    dashboard_profile_url = {}
+    if "ENABLE_REACT_DASHBOARD" in FEATURES and FEATURES["ENABLE_REACT_DASHBOARD"]:
```

Comment:
I didn't know why but `FEATURES` dict did not contain `get` method here.
I had to change my `get` to this in order to make it work.

I'll add the check for joanie :+1:

Reply:
I think that's because you are manipulating a class property and not an instance property 🤔

You could move this stuff within `post_setup` hook if you need to use the instance property.

Reply:
It's not possible to use `is_joanie_enabled` in this `settings.py` file as this template_tag required settings to have been completely loaded.
So i tried to reuse what's done in `is_joanie_enabled` but there again, app arent loaded and it refuse to execute `import_string`

There what didn't work for me : 
```py
any(
    getattr(lms, "is_joanie", False)
    for lms in {
        import_string(lms_configuration["BACKEND"])
        for lms_configuration in RICHIE_LMS_BACKENDS
    }
)
```

Reply:
it did work in `post_setup()` \o/